samples/nrf9160/serial_lte_modem/client/at_client/main.c

```diff
@@ -0,0 +1,316 @@
+/**
```

Comment:
I don't think we should introduce non-NCS samples into NCS codebase. @carlescufi Can you comment on that, was that agreed before?

@junqingzou I see that the sample was completely rewritten, and now supports AT-like commands for custom modules (gps/tcpip etc). We already have an `at_client` sample in NCS, are you familiar with it? Perhaps it would be possible to reuse it's codebase and prepare a NCS version of the client application?

Reply:
Sorry for the misleading name "at_client", which is actually nRF52 project to work with the Serial LTE Modem. I'll remove it from this PR. Serial LTE Modem could work with either PC terminal or MCU code. "at_client" is a sample to run it in MCU code.

Reply:
@rlubos Serial LTE Modem is copied from nrf\lib\at_host then add what PMT asked for. There might be more local AT commands to add based on customer requests.